### PR TITLE
Fix/remove missing package

### PR DESCRIPTION
**Pull request by dohq**
## Description
bootloadhid package is not manage official archlinux repository.
this commit is remove bootloadhid package from `pacman -S` option.
and try to install with yay(AUR package manager).
If yay is not installed, print a message to install bootloadhid manually.

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

NOP

## Checklist
Changed script is only util script.

### CODE REVIEW

**Comment by dohq** [comment]
> The bootloadHid install part at the end of the script currently runs for all linux platforms [/util/linux_install.sh@`master`#L234](https://github.com/qmk/qmk_firmware/blob/master/util/linux_install.sh#L234).
> 
> I think the only thing thats 100% needed is to remove bootloadhid from the pacman install line. Otherwise we need logic within `install_bootloadhid` to see if it already exists somewhere on the path.

gotcha.
I remove install bootloadHID from aur.

File: util/linux_install.sh [hunk 1]
```diff
@@ -12,12 +12,15 @@ util_dir=$(dirname "$0")
 
 # For those distros that do not package bootloadHID
 install_bootloadhid() {
-    wget https://www.obdev.at/downloads/vusb/bootloadHID.2012-12-08.tar.gz -O - | tar -xz -C /tmp
-    cd /tmp/bootloadHID.2012-12-08/commandline/
-    if make; then
-        sudo cp bootloadHID /usr/local/bin
+    type bootloadHID >/dev/null 2>&1
```
Comment (zvecr) [inline util/linux_install.sh]:
```suggestion
```

File: util/linux_install.sh [hunk 2]
```diff
@@ -12,12 +12,15 @@ util_dir=$(dirname "$0")
...
+    if [[ $? -ne 0 ]]; then
```
Comment (zvecr) [inline util/linux_install.sh]:
```suggestion
    if command -v bootloadHid >/dev/null; then
```

**Comment by dohq** [comment]
> Can we align the existence checks to the other util scripts?

OK.
sorry, I didn't check the other scirpts.
commit suggestion.

File: util/linux_install.sh [hunk 3]
```diff
@@ -12,12 +12,14 @@ util_dir=$(dirname "$0")
 
 # For those distros that do not package bootloadHID
 install_bootloadhid() {
-    wget https://www.obdev.at/downloads/vusb/bootloadHID.2012-12-08.tar.gz -O - | tar -xz -C /tmp
-    cd /tmp/bootloadHID.2012-12-08/commandline/
-    if make; then
-        sudo cp bootloadHID /usr/local/bin
+    if command -v bootloadHid >/dev/null; then
```
Comment (zvecr) [inline util/linux_install.sh]: Sorry, got my suggestion wrong on the previous attempt....
```suggestion
    if ! command -v bootloadHID >/dev/null; then
```

Comment (dohq) [reply util/linux_install.sh]: I missed it, too.
thanks

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**